lib/lotus/helpers/html_helper/empty_html_node.rb

```diff
@@ -12,6 +12,12 @@ class EmptyHtmlNode
         # @api private
         ATTRIBUTES_SEPARATOR = ' '.freeze
 
+        # HTML5 form elements where required should be reformated
+        #
+        # @since 0.2.1
+        # @api private
+        HTML5_REQUIRED_CONSTRAINT = %('input' 'select' 'textarea' 'radio').freeze
```

Comment:
In that case i'd rename the constant (and add a freeze to the inner array as well).

I'd suggest to call them "boolean attributes", i've seen this is various places. "disabled" is another one of those.